lib/inline-imports.js

```diff
@@ -59,7 +59,7 @@ function* linesWithInlinedImportsOf(
         },
         visited,
       );
-      yield { line, match: true, lineNumber };
+      yield { line, match: true, lineNumber, resolvedPath };
```

Comment:
`resolvedPath` may be an OK name, but I am curious if there is a better name that is more specific